app/src/main/res/layout/failed_payments_card.xml

```diff
@@ -5,33 +5,47 @@
     android:id="@+id/failedPaymentsCard"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:theme="@style/ThemeOverlay.Hedvig.WarningCard"
     android:layout_marginStart="@dimen/base_margin_double"
     android:layout_marginTop="@dimen/base_margin_double"
     android:layout_marginEnd="@dimen/base_margin_double">
 
-    <LinearLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="@drawable/background_rounded_corners"
-        android:backgroundTint="?colorError"
         android:orientation="horizontal"
         android:padding="@dimen/base_margin_double">
 
         <ImageView
-            android:layout_width="@dimen/icon_failure_small_size"
```

Comment:
Possible compound drawable?

Reply:
🤯 you're on fire with these compound drawables man

Reply:
Actually, come to think of it, it's hard to get the constraints right if using a compound drawable. I'm going to skip it! 🙂